multiple.md

```diff
@@ -0,0 +1,27 @@
+# Handling Multiple Revisions Simultaneously (Example)
```

Comment:
nit: this file name is awkward, consider a longer name.

Reply:
changed to `multiple_revision_example.md`